web/packages/api/src/assets_v2.ts

```diff
@@ -612,6 +612,78 @@ export async function fromContext(context: Context): Promise<RegistryOptions> {
     return result
 }
 
+async function checkSnowbridgeV2Support(
+    parachain: ParachainBase,
+    parachainId: number,
+    info: ChainProperties,
+    hasDryRunApi: boolean,
+): Promise<boolean> {
+    console.log(`Checking Snowbridge V2 support for ${info.specName} (parachain ${parachainId})`)
+    if (!hasDryRunApi) {
+        console.error(`  No dry run API available`)
+        return false
+    }
+
+    try {
+        const testAddress = info.accountType === "AccountId32"
+            ? "0x0000000000000000000000000000000000000000000000000000000000000000"
+            : "0x0000000000000000000000000000000000000000"
+
+        const origin = { system: { signed: testAddress } }
+
+        await parachain.provider.call.dryRunApi.dryRunCall(origin, "0x00", 5)
```

Comment:
I ran this call manually with xcm versions 1 to 6 on asset hub and they all returned OK. So maybe this is a hallucination.